recipes/cutensor/meta.yaml

```diff
@@ -6,29 +6,31 @@ package:
   version: {{ version }}.{{ patch_version }}
 
 source:
-  # TODO(leofang): Add aarm64, win64
+  # TODO(leofang): Add aarm64
```

Comment:
arm link: https://developer.download.nvidia.com/compute/cutensor/1.2.2/local_installers/libcutensor-linux-sbsa-1.2.2.5.tar.gz
ppc64 link: https://developer.download.nvidia.com/compute/cutensor/1.2.2/local_installers/libcutensor-linux-ppc64le-1.2.2.5.tar.gz

Reply:
The ppc64le link was already added in bffa3df, but it won't be packaged until the CUDA support is up (see https://github.com/conda-forge/cupy-feedstock/issues/14#issuecomment-760705486).

Likewise the aarm64 package will also have to wait, but I can add it as a comment in the recipe. 🙂

Reply:
> Likewise the aarm64 package will also have to wait, but I can add it as a comment in the recipe. 🙂

Done in 0e255bb.